spec/dummy/spec/path_converter_spec.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.describe "the path converter in use", type: :request do
+  before do
+    get "/foo/101"
+    get "/foo/102"
+    get "/foo/103"
+    get "/metrics"
+  end
+
+  it "returns metric names using the controller and action used to access the path" do
+    expect(response.body.include?('path="foo#bar"')).to be true
```

Comment:
RSpec has an [include matcher](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher) which you could use here (and for the other tests too). This test would become `expect(response.body).to include('path="foo#bar"')`

Reply:
didn't know about this, thanks :)